README.md

```diff
@@ -1,23 +1,3 @@
 #irods-php
 
-This is a fork of the official DICE-UNC iRODS PHP api.
```

Comment:
i based this PR on https://github.com/mboudet/irods-php/tree/master, because it claimed 4.2 support.
this change is part of 
https://github.com/MaastrichtUniversity/irods-php/commit/44d6c26e8584bb914d2938ebae70a3b2ea179439